.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "standard"
+  "extends": "standard",
+  "parserOptions": {
+    "sourceType": "script"
+  },
+  "rules": {
+    "strict": [
+      "error"
+    ]
+  }
```

Comment:
I was reading last night that you can also put eslint config in `package.json` - perhaps we should do that.